helm-chart/README.md

```diff
@@ -37,8 +37,8 @@ Following are the default configuration values:
 ```yaml
 connection:
   user: postgres
-  password: ""
-  host: db.timescale.svc.cluster.local
+  password: secretPassword
+  host: timescaledb.monitoring.svc.cluster.local
```

Comment:
Let's change this to `timescaledb.default.svc.cluster.local` here and in `values.yaml`. This way we won't be advocating to put timescaledb into `monitoring` namespace.

Reply:
done